packages/vue-components/src/Submenu.vue

```diff
@@ -97,7 +98,7 @@ export default {
     $el.onBlur(() => { this.hideSubmenu(); }, false);
```

Comment:
I think the `onBlur` is not fully working properly? 

Master branch submenu (Notice how the `submenu`s will close when they are blurred.):

https://user-images.githubusercontent.com/47470981/162556008-f5726996-d928-4302-b84d-3ac42e9e3cd5.mp4


Current branch submenu (Multiple `submenu`s of the same dropdown can be opened, resulting the overlapping `submenu`s):

https://user-images.githubusercontent.com/47470981/162556078-f672eed3-cefd-40d3-978e-13fa0a053871.mp4

Reply:
Thank you for the tests @jonahtanjz !